.github/workflows/polyglot_release.yml

```diff
@@ -3,15 +3,20 @@ on:
   push:
     tags:
       - '*'
+  workflow_dispatch:
+    inputs:
+      repository:
+        description: 'PyPI repository (pypi or testpypi)'
+        required: false
+        default: 'pypi'
 jobs:
   release:
     strategy:
       matrix:
         target: [linux-x86, linux-arm, macos-arm]
         python-version: ['3.8', '3.9', '3.10', '3.11', '3.12']
```

Comment:
python 3.8 is EoL'ed, perhaps we can take this opportunity to remove it and add 3.13? https://devguide.python.org/versions/

this can either be bundled in this PR or in a follow-up :)

Reply:
will do this in the next PR once the release is working properly